tripy/tests/test_function_registry.py

```diff
@@ -469,6 +471,30 @@ def func(n: Sequence[Union[int, float]]) -> int:
         ):
             registry["test"](["a", "b", "c"])
 
+    def test_error_variadic_positional_arg_mismatch(self, registry):
+        @registry("test")
+        def func(a: int, *args: int) -> int:
+            return a + sum(args)
+
+        with helper.raises(
+            TripyException,
+            match=dedent(
+                rf"""
+            Could not find an implementation for function: 'test'.
+                Candidate overloads were:
+
+                --> \x1b\[38;5;3m{__file__}\x1b\[0m:[0-9]+ in \x1b\[38;5;6mfunc\(\)\x1b\[0m
+                      \|
+                  [0-9]+ \|         def func\(a: int, \*args: int\) \-> int:
+                  [0-9]+ \|     \.\.\.
+                      \|\s
+
+                Not a valid overload because: For parameter: 'args', expected an instance of type: 'int' but got argument of type: 'str'\.
```

Comment:
Maybe we should just check this part of the message? We have lots of other tests that check how the candidate overloads are displayed.

Reply:
Sure, easy enough.